docs/source/plugin/anatomy.rst

```diff
@@ -252,16 +252,16 @@ The ``shutdown`` function must follow this interface:
 
 .. py:function:: shutdown(bot)
 
-   :param bot: the bot's instance
+   :param bot: the bot instance
    :type bot: :class:`sopel.bot.Sopel`
 
 This function is optional. If it exists, it will be called while the bot
-is shuting down. Note that this normally occurs after closing connection
+is shutting down. Note that this normally occurs after closing connection
 to the server, so the behavior of the messaging functions on the
-:class:`sopel.bot.Sopel` object it's passed is undefined and is likely to fail.
+:class:`sopel.bot.Sopel` object it's passed is undefined and they are likely to fail.
```

Comment:
Well, my suggestion here didn't cause an orphan line, but that's because you just applied the suggestion and didn't re-wrap. 😁

Reply:
I'll, eventually. 😏